src/SFA.DAS.EmployerAccounts.UnitTests/Queries/GetSingleDraftApprenticeship/WhenIGetSingleDraftApprenticeship.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Moq;
+using NUnit.Framework;
+using SFA.DAS.EmployerAccounts.Interfaces;
+using SFA.DAS.EmployerAccounts.Queries.GetSingleDraftApprenticeship;
+using SFA.DAS.Encoding;
+using SFA.DAS.Validation;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using SFA.DAS.NLog.Logger;
+using SFA.DAS.CommitmentsV2.Types.Dtos;
+
+namespace SFA.DAS.EmployerAccounts.UnitTests.Queries.GetSingleDraftApprenticeship
+{
+    public class WhenIGetSingleDraftApprenticeship : QueryBaseTest<GetSingleDraftApprenticeshipRequestHandler, GetSingleDraftApprenticeshipRequest, EmployerAccounts.Queries.GetSingleDraftApprenticeship.GetSingleDraftApprenticeshipResponse>
+    {
```

Comment:
EmployerAccounts.Queries.GetSingleDraftApprenticeship. could be removed

Reply:
done